scripts/ci-rpm-build-test

```diff
@@ -6,9 +6,17 @@
 
 # this script runs as root under docker --privileged
 
+OS_MAJOR=$(grep ^VERSION_ID /etc/os-release | cut -d'=' -f2 | sed 's/\"//gI' | cut -d'.' -f1)
+OS_VERSION=$(grep ^VERSION_ID /etc/os-release | cut -d'=' -f2 | sed 's/\"//gI')
+
 # install dependencies
 dnf install -y rpm-build make yum-utils gcc binutils util-linux-ng which git
 dnf install -y libseccomp-devel cryptsetup
+if [[ $OS_TYPE = "rockylinux" ]] && [ $OS_MAJOR -gt 8 ]; then
+    dnf --enablerepo=crb install -y shadow-utils-subid-devel
+else
+    dnf --enablerepo=powertools install -y shadow-utils-subid-devel
+fi
```

Comment:
You can switch this back to always do `--enablerepo=devel`

Reply:
Done. switched back to devel.